soul-spring-boot-starter/soul-spring-boot-starter-plugin/soul-spring-boot-starter-plugin-httpclient/src/main/java/org/dromara/soul/springboot/starter/plugin/httpclient/HttpClientPluginConfiguration.java

```diff
@@ -110,17 +107,11 @@ public HttpClient httpClient(final HttpClientProperties properties) {
                                     .to(builder::nonProxyHosts);
                         });
                     }
-                    tcpClient = tcpClient.option(ChannelOption.SO_TIMEOUT, 5000);
+                    tcpClient = tcpClient.option(ChannelOption.CONNECT_TIMEOUT_MILLIS, 5000);
```

Comment:
It's already set up [here](https://github.com/dromara/soul/blob/000e54d8169c4db70f6c18048ce67eac07cd4ebf/soul-spring-boot-starter/soul-spring-boot-starter-plugin/soul-spring-boot-starter-plugin-httpclient/src/main/java/org/dromara/soul/springboot/starter/plugin/httpclient/HttpClientPluginConfiguration.java#L90-L95)